### PR TITLE
process: add skaffold build to the CI flow

### PR DESCRIPTION
**Pull request by Shabirmean**

### CODE REVIEW

File: point-of-sale-app/skaffold.yaml [hunk 1]
```diff
@@ -41,5 +41,4 @@ deploy:
     - kubernetes-manifests/api-server-lb.yaml
     - kubernetes-manifests/inventory.yaml
     - kubernetes-manifests/payments.yaml
-# [END anthosbaremetal_point_of_sale_app_skaffold_config_backend]
----
+# [END anthosbaremetal_point_of_sale_app_skaffold_config_backend]
```
Comment (ckim328) [inline point-of-sale-app/skaffold.yaml:44]: is this pr up to date with main? I thought we removed the `---` in https://github.com/GoogleCloudPlatform/point-of-sale/pull/80

Comment (Shabirmean) [reply point-of-sale-app/skaffold.yaml:44]: Ahhh good catch. I have to pull the changes in.

Comment (Shabirmean) [reply point-of-sale-app/skaffold.yaml]: Updated in https://github.com/GoogleCloudPlatform/point-of-sale/pull/82/commits/f3a30a171d641668eb78a766cc33bf6dd2d5f7c6

File: .github/cloudbuild/pr-open-to-main-pos.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  args: [ 'install' ]
+  waitFor: [ '-' ]
+
+# create namespace yaml for the PR
+- id: 'create-namespace.yaml'
+  name: ubuntu
+  entrypoint: bash
+  args:
+  - -c
+  - |
```
Comment (ckim328) [inline .github/cloudbuild/pr-open-to-main-pos.yaml:40]: i see that there is `.github/cloudbuild/namespace.yaml` also in the PR, with the same content as within this step, do we need both?

Comment (Shabirmean) [reply .github/cloudbuild/pr-open-to-main-pos.yaml:40]: Good catch. It was used whilst testing. I removed it in 7958f4b165db26d51478f9e4302b2b77353b2650

**Comment by Shabirmean** [comment]
@ckim328 - I've fixed the comments you left.

**Review by ckim328 — APPROVED** [review]
LGTM!! 
🚀 great job setting this up @Shabirmean!

**Pull request merged**